doc/source/intro.rst

```diff
@@ -83,6 +83,13 @@ The following example imports a CDB file.
 See :class:`.FeFormat` for a list of supported FE formats. Check out the
 :ref:`input_file_for_pyacp` section to see how input files can be created.
 
+.. danger::
+    When working on Windows, be careful of possible 
+    `escape sequences <https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences>`_ in paths that could result in errors.
```

Comment:
```suggestion
    When working on Windows, be careful of backslashes in paths: These may correspond to
    `escape sequences <https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences>`_, resulting in errors.
```

Reply:
I like this version! Should we keep the links to the python documentation explaining the two concepts or should we remove them completely @nshum4 ?